weave/integrations/notdiamond/tracing_test.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
For this and other test files, we recently refactored out test layout. Please move test files to `tests/integrations/notdiamond/*`

Reply:
this will be more obvious once updating from master